app/domain/authentication/webservice.rb

```diff
@@ -13,7 +13,6 @@
 
 module Authentication
   class Webservice < ::Dry::Struct
-    constructor_type :schema
```

Comment:
why was this removed?

Reply:
https://dry-rb.org/gems/dry-struct/0.7/constructor-types/#constructor-types-were-removed-in-v0-5-0-use-code-transform_types-code-and-code-transform_keys-code-as-a-replacement

Reply:
don't we need somthing to replace it?